python_package/trackcpp.i

```diff
@@ -97,6 +98,9 @@ double get_double_max() {
 %apply (double* INPLACE_ARRAY2, int DIM1, int DIM2 ) {
     (double *im_out, int n1_im_out, int n2_im_out)}
 
+// For kicktable_getkicks
+%apply double *OUTPUT { double &hkick__, double &vkick__ };
```

Comment:
Why is this needed? I thought we didn't need this type of declaration for python `float` type.

Reply:
it converts input reference arguments (hkick, vkick) to output arguments, so that they ca be retrieved in python,